src/main/resources/task-actions.story

```diff
@@ -11,4 +11,5 @@ When the user starts a random process
 And the user claims a task
 And the user completes the task
 Then the status of the process is changed to completed
+Then the status of the process is changed to completed by querying
```

Comment:
I would remove the last line as querying query service does not add value to the story in terms of business language. It is more a technical verification for us to add that check against query service, so I would add the query service call check in the step above: "Then the status of the process is changed to completed"

Sounds good?